docs/sequence-diagrams/api-integration.puml

```diff
@@ -58,12 +61,16 @@ group periodic evidence submission
 	BE -> Node: Status: 200, Body: {N}
 
 	Node -> Node: Decrypt(EK_priv, N) -> "another-challenge"
-	Node -> BE: POST /node/evidence, Body: {TPMS_ATTEST(nodeID, "another-challenge", evidence)}
-	BE -> V_EviVfy: POST /session/456, Body: TBD1_MimeType{TPMS_ATTEST(nodeID, "another-challenge", evidence)}
-	V_EviVfy -> V_EviVfy: extract nodeID from TPMS_ATTEST.extraData
+	Node -> Node: TPM2_Quote("another-challenge", PCR23) -> evidence
+	Node -> BE: POST /node/evidence, Body: {nodeID, TPMS_ATTEST("another-challenge", evidence)}
+	BE -> BE: repackage {nodeID, golden} as CoRIM with vnd.enacttrust.tpm-evidence payload
```

Comment:
There is no repackaging to CoRIM when we go through the verification API.  It's binary TPMS_ATTEST with `C-T: vnd.enacttrust.tpm-evidence`

Reply:
vnd.enacttrust.tpm-evidence  is not put in a CoRIM package?

Reply:
no, CoRIM is only consumed by the provisioning API.  The verification API expects TPMS_ATTEST in the body and a `Content-Type` header of `vnd.enacttrust.tpm-evidence`

Reply:
Thank you for clarifying. The C-T should be looking like this, right? 

C-T: _application/vnd+enacttrust , profile: tpm-evidence_

Reply:
in this case the expected C-T is `application/vnd.enacttrust.tpm-evidence`